accessibility-checker-extension/src/ts/options/OptionsApp.tsx

```diff
@@ -150,115 +152,136 @@ class OptionsApp extends React.Component<{}, OptionsAppState> {
+              <main aria-labelledby="options">
+                <h1 id="options">IBM Accessibility Checker options</h1>
 
-              <Dropdown
-                ariaLabel="Deployment selection dropdown"
-                disabled={false}
-                helperText="Rule set deployment"
-                id="archivedRuleset"
-                items={archives}
-                itemToString={(item: any) => (item ? item["name"] : "")}
-                label="Rule set deployment"
-                light={false}
-                titleText=""
-                type="default"
-                selectedItem={selected_archive}
-                onChange={this.handleArchiveSelect}
-              />
-              <p className="op_helper-text">
-                For details on rule set changes between deployments, see{" "}
-                <a href="https://github.com/IBMa/equal-access/releases" target="_blank" rel="noopener noreferred">Release notes</a>.
-              </p>
-              <h3>Supported rule sets</h3>
-              <p>
-                Choose which rule set to use. This will affect the issues
-                detected.
-              </p>
+                <h2>Rule set versions </h2>
 
-              <Dropdown
-                ariaLabel="ruleset selection dropdown"
-                disabled={false}
-                helperText="Select rule set"
-                id="archivedRuleset"
-                items={rulesets}
-                itemToString={(item: any) => (item ? item["name"] : "")}
-                label="Rule set deployment"
-                light={false}
-                titleText=""
-                type="default"
-                selectedItem={selected_ruleset}
-                onChange={this.handleRulesetSelect}
-              />
+                <p>
+                  Choose to always use the latest version of the rule sets, use
+                  the version from a specific date, or try a preview of future
+                  rule sets. By default the latest rule set version is selected.
+                </p>
 
-      {selected_ruleset.description? (<p className="op_helper-text">{selected_ruleset.description}</p>):"" }
-              {show_notif ? (
-                <div className="notification">
-                  <InlineNotification
-                    role="alert"
-                    kind="success"
-                    lowContrast={true}
-                    title="Success"
-                    subtitle=" Your changes have been saved"
-                    className=""
-                    iconDescription="close notification"
-                    onCloseButtonClick={() => {
-                      this.setState({ show_notif: false });
-                    }}
-                  />
-                </div>
-              ) : (
-                ""
-              )}
-              <div className="buttonRow">
-                <Button
+                <Dropdown
+                  ariaLabel="Deployment selection dropdown"
```

Comment:
Why is there both a label and an ariaLabel? Also, you should not put 'dropdown' in the label, much as you wouldn't add 'dropdown' to the visible label. Widget type is conveyed to the screen reader through the widget role.

Reply:
For Type script, it looks label and ariaLabel are both required by Dropdown component ,  removing either one of them will cause compilation issue, even the Carbon's doc saying ariaLabel is optional,
https://react.carbondesignsystem.com/?path=/story/dropdown--default

I know the voice over speaks both arialLabel and label. it is kind of redundant. To work around, I will keep the arialLabel, but assign an empty string to it like this: ariaLabel="". Hope it is OK.

Reply:
@zhenwan just fyi, aria-label overrides the native label, assigning ariaLabel="" is not a good idea.

Reply:
also, the attribute is aria-label and not ariaLabel

Reply:
Using ariaLabel="" it to work around Typescript issue. please see my comment above.

"ariaLabel" is a property of Dropdown component. We can not use ' aria-label'